tests/src/Unit/JwtTranscoderBaseTest.php

```diff
@@ -116,8 +116,8 @@ public function testBadSignature(): void {
     $transcoder = $this->createTranscoder($logger);
     $expire_timestamp = gettimeofday(TRUE) + 55555;
     $token = $transcoder->encode(['a' => 'A'], $expire_timestamp);
-    // Insert a bad character in the signature part.
-    $token[-2] = 'x';
+    // Change one character in the signature part.
+    $token[-2] = ($token[-2] !== 'x') ? 'x' : 'y';
```

Comment:
This was causing random fails.
I think sometimes that character is already 'x' so then the poisoning does not work.